easybuild/easyconfigs/h/Hadoop/Hadoop-2.5.0-cdh5.3.1-native.eb

```diff
@@ -26,7 +26,7 @@ dependencies = [(java, javaver)]
 
 build_native_libs = True
 
-extra_native_libs = [('snappy', 'lib/libsnappy.so')]
+extra_native_libs = [('snappy', 'lib/libsnappy.so*')]
```

Comment:
I actually think the easyblocks mostly work along the wrong axis. There should be blocks for fetching, unpacking, building, installing, testing, etc; and the easyconfigs specify which blocks to assemble and use.

But that's not really the discussion I want to have in a PR where I'm just adding a single character to an existing eb file.